.circleci/config.yml

```diff
@@ -4,16 +4,23 @@ runtest: &runtest
     - run:
         name: Install GPflow
         command: |
-          pip install -U pip
-          pip install -e .
-          pip install -r tests_requirements.txt
+          # Run in a fresh virtual environment, to avoid conflicts with preinstalled packages.
+          virtualenv -p python3.6 .venv
+          source .venv/bin/activate
+          pip install --progress-bar=off -U pip
+          # Ensure consistency between tensorflow and tensorflow-probability
+          pip install --progress-bar=off tensorflow==2.6.2 tensorflow-probability==0.14.0
```

Comment:
Should it really be pinning patch version, or just the minor versions, 2.6 and 0.14 so that we will automatically pull in e.g. tf 2.6.3 or tfp 0.14.1?